packages/assets-query-client/src/al-assets-query-client.ts

```diff
@@ -229,14 +229,16 @@ export class AlAssetsQueryClientInstance {
    * "comment": "This risk is acceptable", "expires": 0,
    * "remediation_ids": ["ids_upgrade_hardware"], "deployment_ids":["0A2DC25F-5B5A-4A93-9413-1A8D6F87489E"], filters:[]}'
    */
-  async disposeRemediations(accountId: string, remediationData: { deployment_ids: string[], operation: string, filters: string[], remediation_ids: string[],
-                                                                  reason: string, comment: string, expires: number}) {
+  async disposeRemediations(accountId: string, remediationData: { deployment_ids?: string[], filters: string[], vulnerability_ids?: string[], remediation_ids?: string[], reason: string,
+                                                                  comment: string, expires?: number}) {
```

Comment:
You need to remove the comment `property` in the remediationData argument. Leaving this in means it has to be supplied which we don't want